static/src/javascripts/boot.js

```diff
@@ -1,5 +1,12 @@
 // @flow
 
+// es7 polyfills not provided by pollyfill.io
+import 'core-js/modules/es7.object.values';
+import 'core-js/modules/es7.object.entries';
```

Comment:
We have this polyfill :-) -- https://github.com/Financial-Times/polyfill-service/tree/a1bd243398089d126151ad5a0000c31437ed68af/polyfills/Object/entries